install-dependencies.sh

```diff
@@ -76,8 +88,11 @@ make WHAT=cmd/kube-apiserver
 cp _output/bin/kube-apiserver $(go env GOPATH)/bin
 cd ..
 
-cd ..
-rm -rf tmp
-
-# KUBEBUILDER_ASSETS has to be a single directory
-cp /usr/bin/etcd $(go env GOPATH)/bin
+# Build etcd
+git clone https://github.com/etcd-io/etcd.git
+cd etcd
+# master doesn't issue appropriate log message to validate etcd startup ("serving insecure client requests on")
+git checkout v3.4.15
+go mod vendor
+./build
+cp ./bin/etcd $(go env GOPATH)/bin
```

Comment:
`$(go env GOPATH)/bin` is used in two places now, could you extract it to variable?